extensions/amp-form/amp-form.md

```diff
@@ -181,3 +182,18 @@ One of the main differences between `:invalid` and `:user-invalid` is when are t
 `.user-valid` and `.user-invalid` classes are a polyfill for the pseudo classes as described above. Publishers can use these to style their inputs and fieldsets to be responsive to user actions (e.g. highlighting an invalid input with a red border after user blurs from it).
 
 See the [full example here](https://github.com/ampproject/amphtml/blob/master/examples/forms.amp.html) on using these.
+
+
+## Security Considerations
+Your XHR endpoints need to follow and implement [CORS Requests in AMP spec](https://github.com/ampproject/amphtml/blob/master/spec/amp-cors-requests.md). 
+
+### Protecting against XSRF
+In addition to following AMP CORS spec, please pay extra attention to [non-idempotent requests note]()
```

Comment:
I think idempotence is the wrong concept here. Maybe just "mutating" or "state changing"